modules/core/02-client/migrations/v7/store.go

```diff
@@ -82,7 +80,7 @@ func handleSolomachineMigration(ctx sdk.Context, store sdk.KVStore, cdc codec.Bi
 		}
 
 		// update solomachine in store
-		clientStore.Set(host.ClientStateKey(), bz)
+		clientKeeper.SetClientState(ctx, clientID, &updatedClientState)
```

Comment:
If `updatedClientState` is used here, is it still necessary to do `bz, err := clienttypes.MarshalClientState(cdc, &updatedClientState)` above?

Reply:
I think @crodriguezvega is right here, we can probably remove `bz, err := ...`

Reply:
Nope, nice catch!